src/matter/cluster/AccessControlCluster.ts

```diff
@@ -80,40 +80,50 @@ export const enum ChangeTypeEnum {
  *
  * @see {@link MatterCoreSpecificationV1_0} § 9.10.5.3
  */
```

Comment:
This is one way to use it: deconstruct tlv into the base classes and reference them directly.